build/chocolatey.mk

```diff
@@ -0,0 +1,30 @@
+#
+# Makefile fragment for Chocolatey actions
+#
+CHOCO         ?= docker run --rm -v $$PWD:$$PWD -w $$PWD linuturk/mono-choco
+
+chocolatey-publish: chocolatey-build
+	@echo "=== $(PROJECT_NAME) === [ choco-publish     ]: publishing chocolatey package"
+	@if [ -z "${CHOCOLATEY_API_KEY}" ]; then \
+		echo "Failure: CHOCOLATEY_API_KEY not set" ; \
+		exit 1 ; \
+	fi
+	@cd newrelic-cli && \
+		$(CHOCO) push --source https://chocolatey.org/ -k ${CHOCOLATEY_API_KEY} newrelic-cli.${PROJECT_VER_TAGGED}.nupkg \
+	; cd ..
+
+
+chocolatey-build:
+	@echo "=== $(PROJECT_NAME) === [ choco-build     ]: building chocolatey package"
+	@if [ -z "${CHOCOLATEY_API_KEY}" ]; then \
+		echo "Failure: CHOCOLATEY_API_KEY not set" ; \
+		exit 1 ; \
+	fi
+	@curl -sL -o newrelic-cli/NewRelicCLIInstaller.msi https://github.com/newrelic/newrelic-cli/releases/download/v${PROJECT_VER_TAGGED}/NewRelicCLIInstaller.msi
```

Comment:
work in `/tmp` instead?  You also added the MSI into git

Reply:
MSI deleted.  Can work in /tmp, but simpler not to.